mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -2088,32 +2088,33 @@ func resourceDataprocClusterRead(d *schema.ResourceData, meta interface{}) error
 	}
 
 	var cfg []map[string]interface{}
+	cfg, err = flattenClusterConfig(d, cluster.Config)
```

Comment:
I don't know if it's actually possible for the `cluster.Config` coming from the API to be `nil`, but if it is, I think `flattenClusterConfig()` will panic. Do you think it would be worth a nil check at the beginning of that function too?

Reply:
Doesn't hurt to check it, I added it :) thanks!